notary-server/tests/integration_test.rs

```diff
@@ -53,7 +58,7 @@ async fn setup_config_and_server(sleep_ms: u64, port: u16) -> NotaryServerProper
         },
         authorization: AuthorizationProperties {
             enabled: false,
-            whitelist_csv_path: None,
+            whitelist_csv_path: "./fixture/auth/whitelist.csv".to_string(),
```

Comment:
Yea sorry I figured the following when I was adding the `enabled` toggle for TLS,

that it's more ergonomic to keep this field as compulsory instead of optional, mainly to prevent people forgetting to add this field when they change from auth-disabled to auth-enabled that can result in runtime error instead of compile-time error